TestProject/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
```

Comment:
Is the test project checked in by mistake?

Reply:
No idea where that came from. Removed.